microsoft-edge/devtools-guide-chromium/coverage/index.md

```diff
@@ -21,22 +21,22 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Find unused JavaScript and CSS code with the Coverage panel
+# Find unused JavaScript and CSS code with the Coverage tool
 
-The **Coverage** panel in Microsoft Edge DevTools helps you find unused JavaScript and CSS code.  Removing unused code may speed up your page load and save your mobile users cellular data.
+The Coverage tool in Microsoft Edge DevTools can help you find unused JavaScript and CSS code.  Removing unused code can speed up your page load and save your mobile users cellular data.  The **Coverage** tool is also called the _Coverage panel_ or _Coverage tab_.
```

Comment:
Can you elaborate on why you added `The **Coverage** tool is also called the _Coverage panel_ or _Coverage tab_`? I don't see this in other pages, why do we need it here?

Reply:
outdated